sdks/wasm/Makefile

```diff
@@ -192,8 +192,14 @@ bcl:
 
 ifdef ENABLE_WASM_NETCORE
 
-NETCOREAPP_VERSION := $(shell cat ../../eng/Versions.props | sed -n 's/.*MicrosoftNETCoreAppVersion>\(.*\)<\/MicrosoftNETCoreAppVersion.*/\1/p' )
-NETCOREAPP_DIR=../../netcore/shared/Microsoft.NETCore.App/$(NETCOREAPP_VERSION)
+#NETCOREAPP_VERSION := $(shell cat ../../eng/Versions.props | sed -n 's/.*MicrosoftNETCoreAppVersion>\(.*\)<\/MicrosoftNETCoreAppVersion.*/\1/p' )
+NETCOREAPP_FEED=https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
+NETCOREAPP_VERSION := 5.0.0-alpha.1.19559.6
+NETCOREAPP_NAME=runtime.webassembly-wasm.Microsoft.Private.CoreFx.NETCoreApp
+NETCOREAPP_DIR=$(NETCOREAPP_NAME).$(NETCOREAPP_VERSION)
+
+$(NETCOREAPP_DIR):
+	nuget install $(NETCOREAPP_NAME) -Version $(NETCOREAPP_VERSION) -Source $(NETCOREAPP_FEED)
```

Comment:
The standalone NuGet may not be available on the machine. It's the reason I added `netcore/corefx-restore.csproj`. Not a blocker but it may come up later once the CI gets wired up.

Reply:
You mean the 'nuget' executable ? Its part of mono.

Reply:
It is, if you have Mono already installed on the machine. That was not the case on many of my CI machines.